flight/hal/drivers/opt4001.py

```diff
@@ -213,7 +207,7 @@ def __init__(
         package: int = 0,
         quick_wakeup: bool = False,
         lux_range: int = 0b1100,
-        conversion_time: int = 0b1000,
+        conversion_time: int = 0b0000,

```

Comment:
no reason to change the driver default values. Instead just input the desired value on instantiation of our devices. Default value should not be the the lowest possible conversion time.

Reply:
Rolling back for now. Waiting time should be adapted based on conversion time